loading.bs

```diff
@@ -409,7 +409,8 @@ An exchange signature is a [=struct=] with the following items:
 The <dfn>signed exchange version</dfn> of a [=response=] |response| is the
 result of the following steps:
 
-1. Let |mimeType| be the [=supplied MIME type=] of |response|.
+1. Let |mimeType| be the result of [=header list/extracting a MIME type=] from
+    |response|'s [=response/header list=].
 1. If |mimeType| is undefined, return undefined.
```

Comment:
It should be either a MIME type or failure at this point.

Reply:
Whoops, thanks!